whitebox/streamlit/tabs/sidebar.py

```diff
@@ -21,48 +22,170 @@ def create_sidebar() -> Tuple[
     Returns:
         Tuple[Union[str, None, bool], bool]: _description_
     """
+    # Setup basic structure parts
     readme = load_config("config_readme.toml")
     center_image()
     st.sidebar.image(load_image("logo.png"), width=120)
     display_links(readme["links"]["repo"])
-
     st.sidebar.write(
         "Whitebox is an open source E2E ML monitoring platform with edge capabilities that plays nicely with kubernetes."
     )
-    st.sidebar.write("Begin immediately by defining the below settings:")
 
-    host_option = st.sidebar.text_input(
-        "Paste here your host url:",
-        max_chars=21,
-        placeholder="http://127.0.0.1:8000",
-        value="http://127.0.0.1:8000",
-    )
+    # Structure sidebar's additional expander sections
+    local_css("whitebox/streamlit/utils/style.css")
+    settings_expander = st.sidebar.expander("Settings", expanded=True)
 
-    api_key_option = st.sidebar.text_input(
-        "Paste here your api key:",
-        max_chars=64,
-        type="password",
-        placeholder="your api key",
-    )
-    setting_confirmation = st.sidebar.checkbox(
-        "I'm ok, proceed with the above settings"
-    )
+    with settings_expander:
+        host_option = st.text_input(
+            "Paste here your host url:",
```

Comment:
I would change this with just `Host URL`

Reply:
ok